tests/unit/dependabot_alerts/core_test.py

```diff
@@ -1,6 +0,0 @@
-from dependabot_alerts.core import hello_world
-
-
-class TestHelloWorld:
-    def test_it(self):
-        assert hello_world() == "Hello, world!"
```

Comment:
The cookiecutter generates a `core.py::hello_world()` placeholder function and a placeholder unittest for it. I just deleted it: I don't think we want to bother with unit tests for `dependabot-alerts`'s core at this time